packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -817,19 +818,24 @@ const CFDPasswordModal = observer(({ form_error, platform }: TCFDPasswordModalPr
             return (
                 <React.Fragment>
                     <Localize
-                        i18n_default_text='Congratulations, you have successfully created your {{category}} <0>{{deriv_keyword}} {{platform}}</0> <1>{{type}} {{jurisdiction_selected_shortcode}}</1> account. '
+                        i18n_default_text='Congratulations, you have successfully created your <0/>{{category}} {{platform}} {{type}} {{jurisdiction_selected_shortcode}} account. '
                         values={{
-                            deriv_keyword: is_eu_user ? 'Deriv' : '',
                             type: accountTypes(),
                             platform:
                                 platform === CFD_PLATFORMS.MT5 ? mt5_platform_label : getCFDPlatformLabel(platform),
                             category: category_label,
                             jurisdiction_selected_shortcode: platform === CFD_PLATFORMS.MT5 ? jurisdiction_label : '',
                         }}
-                        components={[<span key={0} className='cfd-account__platform' />, <strong key={1} />]}
+                        components={[<br key={0} />]}
                     />
                     {platform === CFD_PLATFORMS.DXTRADE || platform === CFD_PLATFORMS.CTRADER ? (
-                        <Localize i18n_default_text='To start trading, transfer funds from your Deriv account into this account.' />
+                        <Localize
+                            i18n_default_text='To start trading, <0 />transfer funds <1 />from your Deriv account into this account.'
+                            components={[
+                                platform === CFD_PLATFORMS.CTRADER && <br key={0} />,
```

Comment:
```suggestion
                                (platform === CFD_PLATFORMS.CTRADER  || platform === CFD_PLATFORMS.DXTRADE) && <br key={0} />,
```

Reply:
The break point if cTrader and Deriv X is different that's why I separated them